lib/Login.dart

```diff
@@ -10,29 +11,51 @@ class Login extends StatefulWidget {
 }
 
 class _LoginState extends State<Login> {
-  var styleInput = const TextStyle(color: Colors.white);
+  var styleInput = const TextStyle(color: Purple);
+  var _isObscure = true;
 
   @override
   Widget build(BuildContext context) {
+    var iconObscure = IconButton(
+      icon: Icon(
+        _isObscure ? Icons.visibility : Icons.visibility_off,
+        color: Purple,
+      ),
+      onPressed: () {
+        setState(() {
+          _isObscure = !_isObscure;
+        });
+      },
+    );
     return Scaffold(
       body: Center(
         child:
             Column(mainAxisAlignment: MainAxisAlignment.spaceEvenly, children: [
+          Row(mainAxisAlignment: MainAxisAlignment.start, children: [
+            SizedBox(width: 15),
+            IconButton(
+              icon: Icon(Icons.arrow_back, color: Purple, size: 30),
+              onPressed: () {
+                Navigator.pop(context);
```

Comment:
You could create a global function to avoid lambda.  Something like: 
`void backButton()`
So it would look like this: 
onPressed: backButton